Add OpInfo for `nn.functional.huber_loss`

ejguan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #67465 Add OpInfo for `nn.functional.cosine_embedding_loss`
* #67464 Add OpInfor for `nn.functional.ctc_loss`
* #67371 Add OpInfo for `nn.functional.poisson_nll_loss`
* **#67376 Add OpInfo for `nn.functional.huber_loss`**
* #67381 Add OpInfo for `nn.functional.hinge_embedding_loss`
* #67356 Add OpInfo for `torch.nn.functional.gaussian_nll_loss`

Differential Revision: [D31974040](https://our.internmc.facebook.com/intern/diff/D31974040)

ejguan committed:
Add OpInfo for `nn.functional.gaussian_nll_loss`

[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `nn.functional.huber_loss`"


Differential Revision: [D31974040](https://our.internmc.facebook.com/intern/diff/D31974040)

[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `nn.functional.huber_loss`"


Differential Revision: [D31974040](https://our.internmc.facebook.com/intern/diff/D31974040)

[ghstack-poisoned]

mruberry reviewed (approved):
LGTM! Rebasing this stack should fix the distributed test failures

ejguan committed:
Update on "Add OpInfo for `nn.functional.huber_loss`"


Differential Revision: [D31974040](https://our.internmc.facebook.com/intern/diff/D31974040)

[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `nn.functional.huber_loss`"


Differential Revision: [D31974040](https://our.internmc.facebook.com/intern/diff/D31974040)

[ghstack-poisoned]
